packages/api/cms-api/src/index.ts

```diff
@@ -69,6 +69,7 @@ export { defaultDamAcceptedMimetypes } from "./dam/common/mimeTypes/default-dam-
 export { DamConfig } from "./dam/dam.config";
 export { DAM_CONFIG, IMGPROXY_CONFIG } from "./dam/dam.constants";
 export { DamModule } from "./dam/dam.module";
+export { EntityInfo, EntityInfoServiceInterface } from "./dam/files/decorators/entity-info.decorator";
```

Comment:
This file shouldn't be located in dam/files I suppose

Reply:
Moved to /dependencies in 6553d558a39968a50339b8ce3fbf1d108286fd8f